### ISSUE TITLE
Animation Editor: Viewport and anim graph window capped at 30 Hz

### ISSUE THREAD

**Issue by amzn-jillich** [o3de/o3de#2464]
While the main editor viewport is running with 60 fps, the animation editor's viewport as well as the anim graph window are capped at 30 fps even with empty views and frame times that should allow much higher fps.

**Expected behavior**
Rendering happens at 60Hz, same as in the main editor viewport.

**Screenshots**
![image](https://user-images.githubusercontent.com/43751992/127121645-c4fd0224-f1a5-4b30-b8ca-215e3edca002.png)

**Desktop/Device:**
 - Device: PC
 - OS: Windows
 - Version 10
 - CPU: AMD Ryzen Threadripper 3970X
 - GPU: NVIDIA GeForce RTX 2080 Ti
 - Memory: 64 GB

**Comment by amzn-jillich** [comment]
Currently, the emfx system component is hooked into the tick bus and tells the editor to render when a tick arrives. Despite from the fact, that the system component controlling this is not the right place, the render function of the render plugin in the animation editor gets called at 60 Hz. I got a frame counter in there and double checked.

The issue seems to inside the GLWidget::Update() function which calls the QOpenGLWidget::update() internally. This function also gets called at 60Hz. QOpenGLWidget::update() calls paintGL() internally and calling update() is the proper way of telling Qt to update the opengl widget. Qt seems to skip some of the calls and "optimizes" them out though. Calling qApp->processEvents(); before the update() increases the call-rate of paintGL() and it runs at 60Hz after. Calling processEvents() is a terrible thing to do and I just did that for testing purposes.

We need to find a way to force update() the widget in order to fix this issue. This happens for both, the 3D viewport as well as the anim graph window. Porting the 3D viewport to Atom, will fix that one, but the anim graph window would still have that issue, so we will need a solution either way.
